src/store/builder/index.ts

```diff
@@ -275,7 +288,18 @@ export const actions: ActionTree<IBuilderState, IRootState> = {
       keepAnimatingIfPersistWasTooFast,
     ])
 
+    if (newFlowContainer === null || newFlowContainer === undefined) {
+      // A console error was logged in the flow/flow_persist, but we need to notify users here
+      Vue.$toast.error(`${Lang.trans('flow-builder.error-while-saving-flow')}`)
```

Comment:
do you know how this would look/behave? Does it have a default timeout or a close btn?

Reply:
we have to redefine the global option in consumer side https://bitbucket.org/voto/%7Be5920ca7-e0cc-4cc7-af7a-574c04a6e31c%7D/pull-requests/3132#Lresources/js/app.tsT242, otherwise it will just have the default behaviour from the author.